sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOReadTest.java

```diff
@@ -591,7 +604,7 @@ public void testBigQueryTableSourceInitSplit() throws Exception {
         ValueProvider.StaticValueProvider.of(table),
         fakeBqServices,
         TableRowJsonCoder.of(),
-        BigQueryIO.TableRowParser.INSTANCE);
+        BigQueryIO.TableRowAvroParser.INSTANCE);
```

Comment:
If this class was on the public API surface of BigQueryIO and wasn't marked `@Experimental`, it can not be renamed.

Reply:
Done.